onnx/backend/test/case/node/__init__.py

```diff
@@ -105,7 +105,7 @@ def _extract_value_info(input, name, ele_type=None):  # type: (Union[List[Any],
     if isinstance(input, list):
         return onnx.helper.make_sequence_value_info(
             name=name,
-            elem_type=onnx.mapping.NP_TYPE_TO_TENSOR_TYPE[input[0].dtype],
+            elem_type=onnx.mapping.PYTHON_TYPE_TO_ELEMENT_TYPE[type(input[0])],
```

Comment:
The existing "make_sequence_value_info" seems to have a limitation, as it assumes that we only allow sequences of tensors and the elem-type parameter seems to be that of the type of elements in the tensor. So, I think this is not going to be consistent with that. We need to either (a) Restrict ourselves to sequences of tensors, in which case we should pick up the element-type of the numpy array here, or (b) Have a more comprehensive type-constructor (which would have to be recursive to handle sequences/maps contained inside sequences).

Reply:
Right now, the implementation is option (a). A future PR will address (b).